Each jobs command is unique, so if the command causes a crash, we'll identify it on a unique random number

flodnv opened the pull request:
@tylerkaraszewski please review

$ https://github.com/Expensify/Expensify/issues/78808

Not tested

coleaeason commented on plugins/Jobs.cpp:
```diff
@@ -44,6 +44,10 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
     const string& requestVerb = request.getVerb();
 
+    // Each command is unique, so if the command causes a crash, we'll identify it on a unique random number.
+    command.request["crashID"] = to_string(SRandom::rand64());
```
I don’t think this is going to work... this is it going to prevent the same bad command from crashing multiple nodes.

coleaeason replied:
Okay so i rethought about this, and i was wrong.

tylerkaraszewski commented:
Seems to break the tests.

flodnv commented:
Updated @tylerkaraszewski, the issue was that I broke the expected tests behavior in the jobs plugin with this PR -- so I updated the test to be more realistic. Let me know what you think 🙇
